src/java/org/apache/cassandra/tools/nodetool/GetInterDCStreamThroughput.java

```diff
@@ -18,16 +18,23 @@
 package org.apache.cassandra.tools.nodetool;
 
 import io.airlift.airline.Command;
-
+import io.airlift.airline.Option;
 import org.apache.cassandra.tools.NodeProbe;
 import org.apache.cassandra.tools.NodeTool.NodeToolCmd;
 
-@Command(name = "getinterdcstreamthroughput", description = "Print the Mb/s throughput cap for inter-datacenter streaming in the system")
+@Command(name = "getinterdcstreamthroughput", description = "Print the Mb/s throughput cap for inter-datacenter streaming and entire SSTable inter-datacenter streaming in the system")
 public class GetInterDCStreamThroughput extends NodeToolCmd
 {
+    @SuppressWarnings("UnusedDeclaration")
+    @Option(name = { "-esst", "--print-entire-sstable-throughput" }, description = "Print entire SSTable streaming throughput")
```

Comment:
The short option names typically have single character. Maybe use `-e` here for the entire file.
For the long option name, `print` seems redundant because the command is already `getinterdcstreamthroughput`.

If you agree, please also update the other commands, `GetStreamThroughput`,  `SetInterDCStreamThroughput` (set in the option name is redundant) and `SetStreamThroughput`.